### ISSUE TITLE
In Modern Warfare II, XeSS Performance Mode runs consistently slower than alternative upscaling methods

### ISSUE THREAD

**Issue by JohnnyTex** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#242]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Call of Duty: Modern Warfare II

### Game Platform [Required]

- [ ] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [X] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

i5 12600k

### Graphic Card [Required]

ARC A770 LE

### GPU Driver Version [Required]

- [X] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034
- [ ] 31.0.101.4032
- [ ] 31.0.101.3975
- [ ] 31.0.101.3959
- [ ] 31.0.101.3802
- [ ] 31.0.101.3793
- [ ] 31.0.101.3491
- [ ] 31.0.101.3490

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[4123ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/10732051/4123ssu.txt)

### Description and steps to reproduce [Required]

Compared to FSR and NIS (AMD and Nvidia's upscaling methods in MW2), XeSS' performance mode is almost always much slower. For example, on Santa Sena XeSS Performance seems to average around 90fps or perhaps 100fps when action isn't heavy, but regular dips into the 80s and even 70s. FSR's performance mode on the other hand regularly stays above 100fps, sometimes gets as high as 120fps. On Farm 18 XeSS Performance gets the average to around 100fps, maybe into the 110s, but still dips into the 90s. FSR Performance, meanwhile, punches well above 120fps on average, and in fact reaches as high as the 160s. The difference isn't *always* that bad, but trust me; compare the two on every map and FSR (and even NIS) always comes out on top, and often by a noticeable margin. And while it's true that XeSS looks noticeably better than FSR, for a "performance" tier of XeSS to trail FSR's "performance" tier doesn't sit well. 

### Game graphic quality [Required]

- [ ] Low
- [ ] Medium
- [ ] High
- [X] Ultra

### Game resolution [Required]

2560 x 1440

### Game VSync [Required]

Off

### Game display mode [Required]

- [ ] Fullscreen
- [ ] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

I'm using the Ultra preset w/ minor tweaks.

**Comment by IntelSupport-Arun** [comment]
Hi @JohnnyTex ,

Thank you for contacting us.
We are checking this issue from our side and will update you soon.

**Comment by IntelSupport-Arun** [comment]
Hi @JohnnyTex ,


There is a new Beta driver version [31.0.101.4125](https://www.intel.com/content/www/us/en/download/729157/intel-arc-iris-xe-graphics-beta-windows.html) has been released. Could you check and confirm the issue is still seeing with the latest driver?

**Comment by JohnnyTex** [comment]
Unfortunately yes, XeSS remains the slowest upscaling method.

**Comment by JohnnyTex** [comment]
I don't think that this has changed in the light of what I've just posted in #246 , but I"ll try again soon and lyk

**Comment by IntelSupport-Arun** [comment]
Hi @JohnnyTex ,

We will verify the issue on our side and let you know the update.

**Comment by Arturo-Intel** [comment]
This case trigger a good discussion in the XeSS and drivers team. We are working now on getting data and verifying why this is behaving this way. 

This will take some time tho, please bear with us. 
I will keep you posted
-- r2

**Comment by Arturo-Intel** [comment]
Hey @JohnnyTex 
The team is still working on this issue, It took some time because the XeSS 1.1 was released and we are waiting for the COD devs team to implemented it, and I am not really sure if they have already do it.

What I notice is that without any upsacaler, running the benchmark in Ultra in 1440p, ARC 770 have around 10 FPS more than the RTX 3060 which is really ironic xD

Be sure that the team is very aware about this problem, we are in constant communication, specially when a new driver comes up. 
So in short:
We are still working in this and i will keep you updated in this thread.

--r2

**Comment by JohnnyTex** [comment]
This is very encouraging to hear! And I'm not surprised. The A770 is better than the 3060.

**Comment by rmorera-intel** [comment]
Hi @JohnnyTex 

Can you re-test with the latest driver 31.0.101.5085? There have been several game updates since our last update and we could use your input.

**Comment by JohnnyTex** [comment]
The update I presume being MW3, which I haven't gotten and probably won't. But I could so some similar testing in Warzone 3 to see if XeSS has sped up to catch up to FSR this weekend.

**Comment by rmorera-intel** [comment]
Sounds good. I'll wait for your feedback.

**Comment by JohnnyTex** [comment]
I'm under the impression _so far_ that XeSS has indeed sped up, but FSR I think still overall holds the lead. This is comparing what both sides call "Performance" Mode. I'd only played a couple of matches, so perhaps further testing will show that it's actually all evened out, but so far I think FSR still has a noticeable lead, but XeSS has caught up substantially. When playing on Ashiika Island for example, the slowest XeSS got around the castle area was into the 70s, while FSR never fell below the 80s. Not an insignificant difference.

**Comment by rmorera-intel** [comment]
Thanks for the feedback. I'll close this bug report now as the majority of the improvements on XeSS for this title will be delivered in a combination of driver and game patches.
